imports/node-app/core/util/defineCollections.js

```diff
@@ -29,7 +29,6 @@ export default function defineCollections(db, collections) {
     ProductSearch: db.collection("ProductSearch"),
     roles: db.collection("roles"),
     SellerShops: db.collection("SellerShops"),
-    Shipping: db.collection("Shipping"),
```

Comment:
Assuming this is just because we are creating `Shipping` inside it's own package?

Reply:
Yep